### ISSUE TITLE
Add web accessibility tools for users

### ISSUE THREAD

**Issue by yucheng11122017** [MarkBind/markbind#2410]
### Please confirm that you have searched existing issues in the repo

Yes, I have searched the existing issues

### What is the area that this feature belongs to?

Reader Usability

### Is your feature request related to a problem? Please describe.

Would be good to include tools to help users enhance the accessibility of their website. 

### Describe the solution you'd like

We can include web accessibility tools to help detect and fix accessibility problems for example Axe as used by Teammates currently. 

### Describe alternatives you've considered

Other accessibility tools could be explored as well!

**Comment by damithc** [comment]
Good suggestion @yucheng11122017 
I guess one thing to ensure first is that MarkBind does not make content 'less accessible' on our own accord e.g., use default color combinations that don't have enough contrast

**Comment by yucheng11122017** [comment]
Thats a good idea Prof! 
Some things from examining our user guide for now 
- Certain formatting results in lack of contrast eg. grey text, inline code, links
- General improper structure of html eg. Page nav is not formatting correct because <ul> contains a <div> than <li> element
- Missing aria-labels in buttons and links
- Missing lang attributes
- Missing alt text in images

For now, it seems the lack of contrast is the biggest violator in general. 
I will make a PR for now just to correct some of these mistakes if possible and then later look into adding web accessibility tools in Markbind for users to use. :)
